package.json

```diff
@@ -118,7 +120,8 @@
     "prismjs": "^1.25.0",
     "nth-check": "^2.0.1",
     "ansi-regex": "^5.0.1",
-    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.1"
+    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.1",
+    "postcss": "8"
```

Comment:
Why not `^8.4.5`?

Reply:
Ok, I see it took the latest version.

Reply:
I didn't look up what the latest version of postcss is, so I was able to just add `"8"` and run `yarn`.

I can change it if you want.

Reply:
It seems that it is ok https://docs.npmjs.com/about-semantic-versioning#using-semantic-versioning-to-specify-update-types-your-package-can-accept but my eye is not used to it 😀 Maybe change, the same with `webpack`.

Reply:
Then will you change this one? https://github.com/iTwin/iTwinUI/blame/main/package.json#L69
![image](https://user-images.githubusercontent.com/9084735/148786638-025690ae-6ad9-417c-b0af-eb3efd283b14.png)

Reply:
lol, ok, fair point 😀